integrations/agntcy-dir/Taskfile.yml

```diff
@@ -170,6 +181,14 @@ tasks:
       - defer: { task: manifests:cleanup }
       - REMOVE_CONTAINERS={{.REMOVE_CONTAINERS}} RUNNER_TYPE={{.RUNNER_TYPE}} PATH=$PATH:{{.DIRCTL_BIN_PATH}} IMAGE_REPO={{.IMAGE_REPO}} DIRECTORY_IMAGE_TAG={{.DIRECTORY_IMAGE_TAG}} go test ./tests -v -failfast -test.v -test.paniconexit0 -ginkgo.timeout 30m -timeout 30m -ginkgo.v -ginkgo.focus "Agntcy agent list tests"
 
+  test:networking:
+    desc: Directory agent networking test
+    cmds:
+      - task: k8s:peers:port-forward:setup
+      - defer: { task: k8s:port-forward:teardown }
+      - defer: { task: manifests:cleanup }
```

Comment:
As I see the `manifests:cleanup` task hasn't been defined. So I think we can remove this.